### PR TITLE
ui redesign

### PR DESCRIPTION
**Pull request by pabloamg**
Hi, I merged some work i had begun for a floatplane app with flutter with your work, let me know what you think

### CODE REVIEW

**Comment by JoeZwet** [comment]
I'll take a look at it when I get home later today.

File: lib/views/browse/browse.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:floatplane/config/ui_data.dart';
+import 'package:floatplane/data_factory/app_state.dart';
+import 'package:floatplane/data_factory/models/creator/creator.dart';
+import 'package:floatplane/views/browse/creator_list.dart';
+import 'package:floatplane/views/browse/drawer_menu.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+
+class Browse extends StatelessWidget {
+  @override
```
Comment (JoeZwet) [inline lib/views/browse/browse.dart]: double override or just typo?

Comment (pabloamg) [reply lib/views/browse/browse.dart]: just typo

**Comment by pabloamg** [comment]
Hi, did you take a look?

**Comment by JoeZwet** [comment]
hey i've merged this but could you explain some of the things you did so i can get a better understanding of the json stuffs in flutter?

**Pull request merged**